modules/deploy/pages/deployment-option/self-hosted/kubernetes/local-guide.adoc

```diff
@@ -501,7 +501,7 @@ kubectl --namespace <namespace> get secret redpanda-external-root-certificate -o
 +
 --
 [loweralpha]
```

Comment:
Out of curiosity what is `[loweralpha]` for?

Reply:
@JakeSCahill